ZenPacks/zenoss/LinuxMonitor/zenpack.yaml

```diff
@@ -200,6 +244,35 @@ device_classes: !ZenPackSpec
                                 dpName: volume_free
                                 lineType: AREA
                                 rpn: "size,-,size,/,-100,*"
+                    Physical Volume IO Stats:
+                        units: per sec
+                        miny: 0
+                        maxy: 100
+                        graphpoints:
+                            RRQM:
```

Comment:
It's probably not reasonable to use RRQM and WRQM as graph legends. I've been working heavily with Linux for decades without ever seeing those acronyms.